packages/relay-runtime/store/RelayReader.js

```diff
@@ -1415,17 +1424,29 @@ function getResolverValue(
 
   let resolverResult = null;
   let resolverError = null;
+
+  const shouldPassContextToLiveResolver = field.kind === 'RelayLiveResolver';
   try {
     const resolverFunctionArgs = [];
     if (field.fragment != null) {
       resolverFunctionArgs.push(fragmentKey);
+    } else {
+      // Set first argument to `null` in case we have resolver context
```

Comment:
Nit: The code is pushing `undefined`, not `null. What gives?

Reply:
I forgot to change the comment, but I changed it to `undefined` to keep consistent with the `: undefined` we will push for the args value